### PR TITLE
the solution of the 'user mapper' problem

### PR DESCRIPTION
**Pull request by koryun23**

### CODE REVIEW

File: src/test/com/aca/homework/week10/mapper/DefaultUserMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.aca.homework.week10.mapper;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class DefaultUserMapperTest {
+    @Test
+    public void testMap(){
+        User user = new User(1, "student", "Lord");
+        UserMapper mapper = new DefaultUserMapper();
```
Comment (gevorgyanarman) [inline src/test/com/aca/homework/week10/mapper/DefaultUserMapperTest.java:10]: please create test subject in before each method

File: src/test/com/aca/homework/week10/mapper/DefaultUserMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        Assertions.assertEquals(user.getFirstName(), mapper.map(user).getFirstName());
```
Comment (gevorgyanarman) [inline src/test/com/aca/homework/week10/mapper/DefaultUserMapperTest.java:11]: mapper.map should be invoked only once

File: src/test/com/aca/homework/week10/mapper/DefaultUserMapperTest.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        Assertions.assertEquals(user.getUsername(), mapper.map(user).getUsername());
```
Comment (gevorgyanarman) [inline src/test/com/aca/homework/week10/mapper/DefaultUserMapperTest.java:12]: assert that the result.getUsername is equal to concrete string value not another variable

**Review by gevorgyanarman — CHANGES_REQUESTED** [review]

**Comment by koryun23** [comment]
modified the unittest

File: src/test/com/aca/homework/week10/mapper/DefaultUserMapperTest.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package com.aca.homework.week10.mapper;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DefaultUserMapperTest {
+    private UserModel testSubject;
```
Comment (gevorgyanarman) [inline src/test/com/aca/homework/week10/mapper/DefaultUserMapperTest.java:8]: the test subject should be UserMapper not a UserModel

**Review by gevorgyanarman — CHANGES_REQUESTED** [review]

**Comment by koryun23** [comment]
done

**Pull request merged**